packs/data/feats.db/terrain-expertise.json

```diff
@@ -18,14 +18,68 @@
         "level": {
             "value": 1
         },
+        "maxTakable": null,
         "prerequisites": {
             "value": [
                 {
                     "value": "trained in Survival"
                 }
             ]
         },
-        "rules": [],
+        "rules": [
+            {
+                "choices": [
+                    {
+                        "label": "Aquatic",
```

Comment:
Localize these. PF2E.SpecificRule.Terrain.Whatever

Reply:
Would it make more sense to have them localized as PF2E.Terrain.Whatever instead? I imagine terrains aren't/won't be limited to a specific feat or rule.

Reply:
Mostly to avoid cluttering the top level, that one may be general enough to justify going in at the top level though.

Reply:
I've put it at the top level, but I am going to sleep right now. If you decide it needs to be put into SpecificRule instead of top level, just let me know and I'll make the modification as soon as possible.